### PR TITLE
SGPC3+SHTC1 Combo Driver

### PR DESCRIPTION
**Pull request by abrauchli**
Check the following:

 - [ ] Breaking changes marked in commit message
 - [x] Changelog updated
 - [x] Code style cleaned (ran `make style-fix`)
 - [ ] Tested on actual hardware

This should make it easier for users to find and for us to maintain. It carries a readme that explains the rationale behind the `svm30_` naming

### CODE REVIEW

File: sgpc3_with_shtc1/sgpc3_with_shtc1_example_usage.c [hunk 1]
```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017, Sensirion AG
```
Comment (rnestler) [inline sgpc3_with_shtc1/sgpc3_with_shtc1_example_usage.c]: Maybe update for the current year.

File: sgpc3_with_shtc1/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# SGPC3 with SHTC1: A Humidity-compensated Combo Driver for SGPC3
+This folder contains a humidity compensated combo-driver combining the SHTC3 and
+the SHTC1 (family compatible like SHTW1, SHTW2, SHTC3) driver into one with the
+necessary bits for the humidity compensation of the gas signal. It is based on
+the SVM30 driver (hence the name.)
```
Comment (rnestler) [inline sgpc3_with_shtc1/README.md]: I find that confusing if it is also named SVM30.

File: sgpc3_with_shtc1/svm30.c [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    //    return (ret * (*humidity)) / 100000;
+    // Maximum ret = 198277 (Or last entry from AH_LUT_100RH)
+    // Maximum *humidity = 119000 (theoretical maximum)
+    // Multiplication might overflow with a maximum of 3 digits
+    // Trick: ((ret >> 3) * (uint32_t)(*humidity)) does never overflow
+    // Now we only need to divide by 12500, as the tripple righ shift
+    // divides by 8
+
+    return ((ret >> 3) * (uint32_t)(*humidity)) / 12500;
+}
```
Comment (rnestler) [inline sgpc3_with_shtc1/svm30.c]: We should really move this function into `embedded-common` or `embedded-sht`.

File: sgpc3_with_shtc1/README.md (hunk 2 above)
Comment (abrauchli) [reply sgpc3_with_shtc1/README.md]: Fixed

File: sgpc3_with_shtc1/svm30.c (hunk 3 above)
Comment (abrauchli) [reply sgpc3_with_shtc1/svm30.c]: I'd rather do that in a separate commit if you don't mind

Comment (rnestler) [reply sgpc3_with_shtc1/svm30.c]: Sure. Was more meant as a "in the long term" comment.

File: sgpc3_with_shtc1/README.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# SGPC3 with SHTC1: A Humidity-compensated Combo Driver for SGPC3
+This folder contains a humidity compensated combo-driver combining the SHTC3 and
+the SHTC1 (family compatible like SHTW1, SHTW2, SHTC3) driver into one with the
```
Comment (rnestler) [inline sgpc3_with_shtc1/README.md]: I'd either add SHTC1 to the list as well or leave of SHTW1 since it is a legacy device.

Comment (abrauchli) [reply sgpc3_with_shtc1/README.md]: done

**Pull request merged**